changelogs/current.yaml

```
@@ -128,6 +128,10 @@ bug_fixes:
     the number of requests per I/O cycle is configured and an HTTP decoder filter that pauses filter chain is present. This behavior
     can be reverted by setting the runtime guard ``envoy.reloadable_features.use_filter_manager_state_for_downstream_end_stream``
     to false.
+- area: upstream
+  change: |
+    Fixed a bug using hard coded drop_category when reporting drop_overload stats to the load report service.
+    It is changed to use drop_category in the EDS config update message.
```

```suggestion
    It is changed to use drop_category that is set in the ClusterLoadAssignment.Policy.DropOverload.category
```

EDS is not really necessary here.
Please also add a link to the field of `ClusterLoadAssignment.Policy.DropOverload.category`.